src/neoTrackerPanel.ts

```diff
@@ -165,9 +167,33 @@ export class NeoTrackerPanel implements INeoSubscription, INeoStatusReceiver {
                     ((this.viewState.currentBlock !== undefined) ? ActivePage.BlockDetail : ActivePage.Blocks );
             } else if (message.e === trackerEvents.Copy) {
                 await vscode.env.clipboard.writeText(message.c);
+            } else if (message.e === trackerEvents.Search) {
+                const asNumber = parseInt(message.c);
+                const block = await this.rpcConnection.getBlock((asNumber + '') === message.c ? asNumber : message.c, this);
+                if (block) {
+                    this.viewState.currentBlock = block;
+                    this.viewState.activePage = ActivePage.BlockDetail;
+                } else {
+                    const transaction = await this.rpcConnection.getTransaction(message.c, this);
+                    if (transaction) {
+                        this.viewState.currentTransaction = transaction;
+                        this.viewState.activePage = ActivePage.TransactionDetail;
+                    } else if (wallet.isAddress(message.c)) {
```

Comment:
It feels like we should do this check first. I understand that we can't know if a provided hash is a block or transaction without checking. But if the user enters an address, should we also check first to see if it's a block or tx hash?